bundle/run/job_options.go

```diff
@@ -0,0 +1,127 @@
+package run
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	flag "github.com/spf13/pflag"
+)
+
+// JobOptions defines options for running a job.
+type JobOptions struct {
+	// Task parameters are specific to the type of task.
+	dbtCommands       []string
+	jarParams         []string
+	notebookParams    map[string]string
+	pipelineParams    map[string]string
+	pythonNamedParams map[string]string
+	pythonParams      []string
+	sparkSubmitParams []string
+	sqlParams         map[string]string
+
+	// Job parameters are a map of key-value pairs that are passed to the job.
+	// If a job uses job parameters, it cannot use task parameters.
+	// Also see https://docs.databricks.com/en/workflows/jobs/settings.html#add-parameters-for-all-job-tasks.
+	jobParams map[string]string
+}
+
+func (o *JobOptions) Define(fs *flag.FlagSet) {
+	// Define task parameters flags.
+	fs.StringSliceVar(&o.dbtCommands, "dbt-commands", nil, "A list of commands to execute for jobs with DBT tasks.")
```

Comment:
Would it be possible to hide all these options from help? Making them show up only when specifying an extra option?

Reply:
Something for a followup ticket?